docs/organizations/segments.md

```diff
@@ -0,0 +1,37 @@
+# Segments
+Segments are dimensions that Codacy reads from your provider that organizes repositories into relevant groups for you. Today, Segments is available for: [GitHub Custom Properties](#github-custom-properties)
+  
+<br><br>
+### How can Segments be utilised?
+- [Repository list](../managing-repositories/#provider-segments) 
+- [Security & Management Risk](../managing-security-and-risk/)
+
+#### Enabling Segments
+To enable **Segments**, an initial sync between your provider and Codacy needs to happen. Once completed, you can use **Segments** to better locate and organize repositories within Codacy.
+<br>
+
+#### Keep Segments in sync
+For changes to repository **Custom Properties** in GitHub to be **automatically** reflected in Codacy, users need to [accept the new permission request made by the Codacy GitHub app](https://docs.github.com/en/apps/using-github-apps/approving-updated-permissions-for-a-github-app). Accepting this permission enables automatic synchronization of any future changes to the custom properties in GitHub.
+
+!!! note
+    If the permission is **not accepted**, users will still be able to use Repository Custom Properties as filters in Codacy, but will need to manually trigger a sync. This can be done using the **manual sync** button available in the filter dropdown, which allows users to synchronize changes from GitHub, though the process may take longer.
+<br><br>
+
+### GitHub Custom Properties {: id="github-custom-properties"}
+Create, use, and manage custom properties for your repositories directly in GitHub. Custom properties allow you to assign tags or metadata to repositories, making it easier to categorize and filter them.
+
+> ###### How to setup _Custom Properties_ in GitHub
+> ###### For detailed steps on how to add, edit, and manage repository _Custom Properties_ in GitHub, refer to [GitHub's official documentation](https://docs.github.com/en/organizations/managing-organization-settings/managing-custom-properties-for-repositories-in-your-organization#adding-custom-properties).
+
+<br><br>
+***
```

Comment:
What is this whole area? 

It seems to be breaking the UI and it doesn't exist in other pages.

![Screenshot 2024-10-09 at 11 28 56](https://github.com/user-attachments/assets/be6f4098-892d-4bc4-ac2b-31d3ab0ec792)

Reply:
@RaquelHipolito this one wasn't addressed in the most recent changes.